### PR TITLE
Flatten buttons, support any label for dual button, fix dual hover, fix unbranded wide content

### PR DESCRIPTION
**Pull request by bluepnume**
screen shot 2017-08-22 at 8 44 23 pm
screen shot 2017-08-22 at 8 46 45 pm
screen shot 2017-08-22 at 8 48 36 pm

### CODE REVIEW

File: src/components/button/templates/component/style.js [hunk 1]
```diff
@@ -132,181 +128,195 @@ export let componentStyle = `
...
-            border-radius: 15px;
-            font-size: 12px;
+            height: 32px;
+            border-radius: 16px;
         }
 
-        .paypal-button .paypal-button-content .logo.logo-pp {
-            height: 25px;
+        .paypal-button .paypal-button-content,
+        .paypal-button .paypal-button-content .paypal-tagline {
```
Comment (vishakha94) [inline src/components/button/templates/component/style.js]: ` .paypal-button .paypal-button-tag-content {}`
Class name in `template.js` is paypal-button-tag-content for the tag line

File: src/components/button/templates/component/style.js [hunk 2]
```diff
@@ -132,181 +128,195 @@ export let componentStyle = `
...
         .paypal-button .paypal-button-content {
-            height: 24px;
-            max-height: 24px;
-            border-radius: 12px;
+            height: 26px;
+            border-radius: 13px;
+        }
+
+        .paypal-button .paypal-button-content,
+        .paypal-button .paypal-button-content .paypal-tagline {
```
Comment (vishakha94) [inline src/components/button/templates/component/style.js]: ` .paypal-button .paypal-button-tag-content {}`
Class name in `template.js` is paypal-button-tag-content for the tag line

File: src/components/button/templates/component/style.js [hunk 3]
```diff
@@ -132,181 +128,195 @@ export let componentStyle = `
...
-            border-radius: 20px;
-            font-size: 14px;
+            height: 42px;
+            border-radius: 21px;
         }
 
-        .paypal-button .paypal-button-content .logo.logo-pp {
-            height: 30px;
+        .paypal-button .paypal-button-content,
+        .paypal-button .paypal-button-content .paypal-tagline {
```
Comment (vishakha94) [inline src/components/button/templates/component/style.js]: ` .paypal-button .paypal-button-tag-content {}`
Class name in `template.js` is paypal-button-tag-content for the tag line

File: src/components/button/templates/component/template.js [hunk 4]
```diff
@@ -69,27 +51,106 @@ export function componentTemplate({ props } : { props : Object }) : string {
...
+        return buttonTemplate({
+            label: button.label,
+            color: button.color,
+            isDual: Boolean(dual),
+            locale,
+            branding
+        });
+    }).join('\n');
+
+    let taglineHTML = taglineTemplate({
```
Comment (vishakha94) [inline src/components/button/templates/component/template.js]: Do we need to call function `taglineTemplate` if `tagline = false` ?
Instead:
`let taglineHTML = tagline ? taglineTemplate({...}) : ''`

File: src/components/button/templates/component/style.js (hunk 1 above)
Comment (bluepnume) [inline src/components/button/templates/component/style.js]: Good catch!

File: src/components/button/templates/component/style.js (hunk 2 above)
Comment (bluepnume) [inline src/components/button/templates/component/style.js]: 👍

File: src/components/button/templates/component/template.js (hunk 4 above)
Comment (bluepnume) [inline src/components/button/templates/component/template.js]: Good idea 👍

**Comment by bluepnume** [comment]
@vishakha94 updated code, lemme know if you have any comments!

File: src/components/button/templates/component/style.js [hunk 5]
```diff
@@ -235,199 +256,95 @@ export let componentStyle = `
...
-        background: linear-gradient(to bottom, #f5f5f5, #cccccc) 0 100%;
-    }
-
 
 
 
      /* Credit Button */
 
-    .paypal-button.paypal-style-credit .paypal-button-content.paypal-color-creditblue {
+    .paypal-button .paypal-button-content.paypal-label-${ BUTTON_COLOR.GOLD }.paypal-color-${ BUTTON_COLOR.CREDITBLUE } {
```
Comment (vishakha94) [inline src/components/button/templates/component/style.js]: `.paypal-button .paypal-button-content.paypal-label-${ BUTTON_LABEL.CREDIT }.`

File: src/components/button/templates/component/style.js [hunk 6]
```diff
@@ -100,81 +109,96 @@ export let componentStyle = `
...
 
-    .paypal-button .paypal-button-tag-content.paypal-tagline-color-blue {
+    .paypal-button .paypal-tagline.paypal-tagline-color-${ TAGLINE_COLOR.BLUE } {
         color: #003366;
     }
 
-    .paypal-button .paypal-button-tag-content.paypal-tagline-color-black {
-        color: #000;
+    .paypal-button .paypal-tagline.paypal-tagline-color-${ TAGLINE_COLOR.BLACK } {
+        color: #6C7378;
```
Comment (vishakha94) [inline src/components/button/templates/component/style.js]: Is this color a PayPal brand color? Looks a little lighter than the current tagline for the black button.

Comment (bluepnume) [inline src/components/button/templates/component/style.js]: Yeah this is what Nidaul recommended.

File: src/components/button/templates/component/style.js (hunk 5 above)
Comment (bluepnume) [inline src/components/button/templates/component/style.js]: Oh damn, good catch!

File: src/components/button/templates/component/template.js (hunk 4 above)
Comment (bluepnume) [inline src/components/button/templates/component/template.js]: I added the check to the function itself, so we can contain all the logic there.

**Pull request closed without merging**